Example/Tests/Clients/MHVPersonClientTests.m

```diff
@@ -0,0 +1,476 @@
+//
+// MHVPersonClientTests.m
+// MHVLib
+//
+// Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import <XCTest/XCTest.h>
+#import "MHVCommon.h"
+#import "MHVMethod.h"
+#import "MHVPersonClient.h"
+#import "MHVConnectionProtocol.h"
+#import "MHVServiceResponse.h"
+#import "MHVApplicationSettings.h"
+#import "MHVHttpServiceResponse.h"
+#import "MHVErrorConstants.h"
+#import "MHVPersonInfo.h"
+#import "MHVGetAuthorizedPeopleResult.h"
+#import "MHVGetAuthorizedPeopleSettings.h"
+#import "Kiwi.h"
+
+static NSString *kHealthVaultErrorXml = @"<response><status><code>3</code><error><message>Test Error.</message></error></status>";
+
+SPEC_BEGIN(MHVPersonClientTests)
+
+describe(@"MHVPersonClient", ^
+{
+    __block NSString *responseString;
+    __block MHVApplicationSettings *returnedSettings;
+    __block MHVPersonInfo *returnedPersonInfo;
+    __block MHVGetAuthorizedPeopleResult *returnedAuthorizedPeople;
+    __block MHVRecordCollection *returnedRecords;
+    __block NSError *returnedError;
+    
+    KWMock<MHVConnectionProtocol> *mockConnection = [KWMock mockForProtocol:@protocol(MHVConnectionProtocol)];
+    [mockConnection stub:@selector(executeHttpServiceOperation:completion:) withBlock:^id(NSArray *params)
+     {
+         MHVHttpServiceResponse *httpServiceResponse = [[MHVHttpServiceResponse alloc] initWithResponseData:[responseString dataUsingEncoding:NSUTF8StringEncoding]
+                                                                                                 statusCode:0];
+         
+         MHVServiceResponse *serviceResponse = [[MHVServiceResponse alloc] initWithWebResponse:httpServiceResponse isXML:YES];
+         
+         
+         void (^completion)(MHVServiceResponse *_Nullable response, NSError *_Nullable error) = params[1];
+         completion(serviceResponse, serviceResponse.error);
+         return nil;
+     }];
+    
+    let(personClient, ^
+        {
+            return [[MHVPersonClient alloc] initWithConnection:mockConnection];
+        });
+    
+    beforeEach(^{
+        returnedSettings = nil;
+        returnedPersonInfo = nil;
+        returnedRecords = nil;
+        returnedError = nil;
+    });
+    
+#pragma mark - GetApplicationSettings
+    
+    context(@"GetApplicationSettings success", ^
```

Comment:
Same feedback here about BDD test strings. The test name should end up being an English-Like sentence.